honeycomb/honeycomb.go

```diff
@@ -423,6 +423,28 @@ func NewExporter(config Config, opts ...ExporterOption) (*Exporter, error) {
 	}, nil
 }
 
+// Consume from the response queue, calling the onError callback when errors
```

Comment:
See #71 for appeasing _golint_'s complaint about this comment not starting with the method name.